Classes/Private/Manager/Objects/OCTCall.m

```diff
@@ -55,41 +35,4 @@ - (NSUInteger)hash
     return [self.chat.uniqueIdentifier hash];
 }
```

Comment:
I think it is enough to have uniqueIdentifier comparisment. With this method getOrCreateCall... you always will have only one call for chat